src/core/api/robot.go

```diff
@@ -68,15 +67,15 @@ func (r *RobotAPI) Prepare() {
 		return
 	}
 	r.project = project
+	r.ctr = robot.NewAPIController(robot.NewDefaultRobotAccountManager())
```

Comment:
How about declare a global variable for the robot account controller in the controller package? So that we don't need to create a new controller for every request

Reply:
done